gubernator/gcs_async.py

```diff
@@ -36,7 +36,12 @@ def get(url):
         if status in (200, 206):
             content = result.content
             if result.headers.get('content-encoding') == 'gzip':
-                content = zlib.decompress(result.content, 15 | 16)
+                dec = zlib.decompressobj(15 | 16)
+                content = dec.decompress(result.content, MAX_SIZE)
```

Comment:
Also isn't it more common that I care about the end of the log than the beginning?

Reply:
I think the important part for now is not exceeding memory limits / wasting time loading ridiculously large files, we can point users to the full log file.

Reply:
Sure, agreed. I don't know this API -- is it easy to grab the end? If not, let's do this in a follow-up.

Reply:
We'd have to decompress into a buffer and discard chunks until we reach the end. Highlighting build logs is a best-effort/heuristic thing, so I'm not sure it's worth it for this one weird user.